src/app/integrations/xero/xero.component.ts

```
@@ -98,6 +100,7 @@ export class XeroComponent implements OnInit {
           () => this.setupWorkspace()
         );
       } else {
+        this.authService.updateUserTokens('XERO')
         this.setupWorkspace();
```

ESLint

[error] Missing semicolon.

(semi)